src/createServer.js

```diff
@@ -1,3 +1,65 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const normalizedUrl = new URL(req.url, `http://${req.headers.host}`);
+
+    res.setHeader('Content-Type', 'application/json');
+
+    const { pathname, searchParams } = normalizedUrl;
+
+    const originalText = pathname.slice(1);
+    const caseParams = searchParams.toString().split('=');
+    const targetCase = caseParams[1];
+    const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
```

Comment:
move it to the variables file

Reply:
Hi, I see you haven't fixed previous comments, maybe you did not push your changes?
If you have any questions, please ask them in fe_chat and tag mentors. It will be faster because your answers in GitHub will be visible to the mentor only after you re-request review 🙂